lib/view/formulier/InvoerVelden.class.php

```diff
@@ -838,7 +838,8 @@ public function getHtml() {
 		$html = '<input name="' . $this->name . '_show" value="' . $show_value . '" origvalue="' . $this->show_value . '"' . $this->getInputAttribute(array('type', 'id', 'class', 'disabled', 'readonly', 'maxlength', 'placeholder', 'autocomplete')) . ' />';
 
 		// actual values
-		$class = $this->model->orm;
+        $model = $this->model;
```

Comment:
Maar waarom werkt dat niet?
```$this->orm->getTableName()``` werkt wel in ForumModel.

Ik vermoed dat dit verband heeft met bovenstaande en dat loadProperty het misschien niet doet. Klopt dat? @qurben